src/uproot_browser/__main__.py

```diff
@@ -28,7 +34,7 @@ def main() -> None:
 
 
 @main.command()
-@click.argument("filename")
+@click.argument("filename", type=click.Path(exists=True))
```

Comment:
Ahh, missed these. It's not correct, this is a path maybe followed by aa colon then stuff.